docs/source/llms/llm-evaluate/index.rst

```diff
@@ -0,0 +1,521 @@
+                "MLflow is an open-source platform for managing the end-to-end machine learning (ML) lifecycle. "
+                "It was developed by Databricks, a company that specializes in big data and machine learning solutions. "
+                "MLflow is designed to address the challenges that data scientists and machine learning engineers "
+                "face when developing, training, and deploying machine learning models.",
+                "Apache Spark is an open-source, distributed computing system designed for big data processing and "
+                "analytics. It was developed in response to limitations of the Hadoop MapReduce computing model, "
+                "offering improvements in speed and ease of use. Spark provides libraries for various tasks such as "
+                "data ingestion, processing, and analysis through its components like Spark SQL for structured data, "
+                "Spark Streaming for real-time data processing, and MLlib for machine learning tasks",
+            ],
+            "predictions": [
+                "MLflow is an open-source platform that provides handy tools to manage Machine Learning workflow "
+                "lifecycle in a simple way",
+                "Spark is a popular open-source distributed computing system designed for big data processing and analytics.",
+            ],
+        }
+    )
+
+    with mlflow.start_run() as run:
+        results = mlflow.evaluate(
+            data=eval_data,
+            targets="ground_truth",
+            predictions="predictions",
+            extra_metrics=[mlflow.metrics.answer_similarity()],
+            evaluators="default",
+        )
+        print(f"See aggregated evaluation results below: \n{results.metrics}")
+
+        eval_table = results.tables["eval_results_table"]
+        print(f"See evaluation table below: \n{eval_table}")
+
+
+View Evaluation Results
+========================
+
+View Evaluation Results via Code
+-----------------------------------
+
+``mlflow.evaluate()`` returns the evaluation results as an :py:func:`mlflow.models.EvaluationResult` instace. 
+To see the score on selected metrics, you can check:
+
+* ``metrics``: stores the aggregated results, like average/variance across the evaluation dataset. Let's take a second
+  pass on the code example above and focus on printing out the aggregated results.
+  
+  .. code-block:: python
+
+    with mlflow.start_run() as run:
+        results = mlflow.evaluate(
+            data=eval_data,
+            targets="ground_truth",
+            predictions="predictions",
+            extra_metrics=[mlflow.metrics.answer_similarity()],
+            evaluators="default",
+        )
+        print(f"See aggregated evaluation results below: \n{results.metrics}")
+* ``tables``: stores the per-row evaluation results. 
+  TODO(prithvi): The code example always uses "eval_results_table" as the key to fetch eval table, is this a hard requirement?
```

Comment:
yes. the table produced by `mlflow.evaluate` will be called `eval_results_table`